src/Umbraco.Web/Editors/EntityController.cs

```diff
@@ -382,6 +383,53 @@ public IEnumerable<EntityBasic> GetChildren(int id, UmbracoEntityTypes type)
             return GetResultForChildren(id, type);
         }
 
+        public PagedResult<EntityBasic> GetPagedChildren(
+            int id,

```

Comment:
I think it's the id parameter that is causing the bad request we see when using the endpoint with members - guessing it's because members aren't using the and integer as an id but an alias or guid or something.